syntax="proto3";
message RequestMsg
{
	int32 cmdType=1;
	string clientId=2;
	string serverId=3;
	bytes sign=4;
	bytes data=5;
}